src/caretogether-pwa/src/Referrals/PartneringFamilyScreen.tsx

```diff
@@ -80,6 +82,68 @@ export function PartneringFamilyScreen() {
     ? `${partneringFamily.family?.adults!.filter(adult => adult.item1!.id === partneringFamily.family!.primaryFamilyContactPersonId)[0]?.item1?.lastName} Family`
     : "...");
 
+  //#region Arrangement Filtering
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+  const handleFilterClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleFilterClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? 'Filter' : undefined;
```

Comment:
I would recommend renaming `open` and `id` to be a bit more clear in the context of this (admittedly) way-too-large function, but I'm half-guessing that you're thinking about extracting the filter to its own component? That would be better (also because then we're starting to slim down the `PartneringFamilyScreen`).

Reply:
This is now fixed & also moved to it's own component over @ FilterMenu.tsx